pkg/registry/apis/provisioning/export.go

```diff
@@ -58,10 +59,10 @@ func (c *exportConnector) Connect(ctx context.Context, name string, opts runtime
 			responder.Error(apierrors.NewBadRequest(err.Error()))
 			return
 		}
-
-		job, err := c.jobs.Add(ctx, &provisioning.Job{
+		job, err := c.jobs.Insert(ctx, &provisioning.Job{
 			ObjectMeta: v1.ObjectMeta{
 				Namespace: cfg.Namespace,
+				Name:      fmt.Sprintf("%s-%s", cfg.Name, provisioning.JobActionExport),
```

Comment:
Cannot we centralize the naming within the jobs instead of passing it from the caller? It seems we do the same `repo + type`.